spec/queries/customers_with_balance_spec.rb

```diff
@@ -18,6 +18,23 @@
       customers_with_balance.query
     end
 
+    describe 'arguments' do
+      context 'with customers collection' do
+        it 'returns balance' do
+          customers = create_pair(:customer)
+          id_b = described_class.new(Customer.where(id: customers)).query
+            .map{ |c| [c.id, c.balance_value] }
+          expect(id_b).to eq([[customers.first.id, 0], [customers.second.id, 0]])
```

Comment:
Thank you for looking through this and your comments! Maybe it was a bit quick to approve it since you still had some legit comments here. Appreciate the idea of focusing on the positive though smile

Anyways I added two commits to address your feedback. Since they don't untouch previous history I'm leaving them separated. Let me know if you would prefer squashing them in the end.

Reply:
Thank you for looking through this and your comments! Maybe it was a bit quick to approve it since you still had some legit comments here. Appreciate the idea of focusing on the positive though :smile: 

Anyways I added two commits to address your feedback. Since they don't untouch previous history I'm leaving them separated. Let me know if you would prefer squashing them in the end.